muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/guider/body/application/current-student/workspaces/workspace.tsx

```diff
@@ -120,23 +118,23 @@ class StudentWorkspace extends React.Component<StudentWorkspaceProps, StudentWor
   }
   render(){
     let workspace = this.props.workspace;
-    
+
     let stateText;
     let extraClasses = "";
     switch (workspace.studentActivity.assessmentState.state){
       case "pending":
       case "pending_pass":
       case "pending_fail":
         stateText = "plugin.guider.assessmentState.PENDING";
-        extraClasses = "state-PENDING";
+        extraClasses = "state-pending";
```

Comment:
Shouldn't we use caps here as in other cases too?

In current-record.tsx file we use state-PASSED etc. but here we use lower case.